script.js

```diff
@@ -81,12 +83,51 @@ function checkWin() {
   return false;
 }
 
+function checkDraw() {
+  if (
+    gameSpaces[0].innerHTML !== "" &&
+    gameSpaces[1].innerHTML !== "" &&
+    gameSpaces[2].innerHTML !== "" &&
+    gameSpaces[3].innerHTML !== "" &&
+    gameSpaces[4].innerHTML !== "" &&
+    gameSpaces[5].innerHTML !== "" &&
+    gameSpaces[6].innerHTML !== "" &&
+    gameSpaces[7].innerHTML !== "" &&
+    gameSpaces[8].innerHTML !== ""
+  ) {
+    return true;
+  }
+  return false;
+}
+
+function disableGameSpace(trueFalse) {
+  gameSpaces.forEach((space) => (space.disabled = trueFalse));
+}
+
+function clearGameBoard(event) {
+  if (text.innerHTML === "PLAYER " + xGamePiece + " WINS!") {
+    text.innerHTML = "Player X turn";
+  }
+  if (text.innerHTML === "PLAYER " + oGamePiece + " WINS!") {
+    text.innerHTML = "Player O turn";
+  }
+  if (text.innerHTML === "DRAW!") {
```

Comment:
you could add this as an OR for the top `if` statement instead of having it as a separate `if` statement since they both result in the same code being run